### DISCUSSION TITLE
Feature Request: Change the Form Submissions UI to improve UX (Similar to Lenix)

### DISCUSSION THREAD

**Discussion by dasignlady** [elementor/elementor#15076] [Feature Request]
Would you consider reworking the form submissions to have a similar UX to Lenix? I wanted to eliminate the Lenix plugin as soon as Elementor form submissions was available, but I like the Lenix interface much better. 

When you click on a form in Lenix, all lead submissions for that form are readable at once.

In Elementor I have to click on each individual lead before I can see the content of the submission. This is a time consuming extra step.

Being able to downlaod CSV is handy but I like to bulk trash spam leads before downloading.

Which brings me to the other point of... why don't Elementor forms include a max character or word limit in text area? I'm sick of Eric Jones spamming my client sites built with Elementor. 

Honeypot and Captcha are worthless against such drivel.

Thanks for your time.

**Comment by nicholaszein** [comment]
Hello @dasignlady,

thanks for submitting your feature request! 🙏 

Please understand that while we do welcome new feature requests, we can't promise they will be considered or added to our roadmap for the time being. 

🔖 We will, of course, keep this suggestion in mind and consider adding it if there is significant demand in the future.

Best regards
